docker-compose.yaml

```diff
@@ -5,3 +5,5 @@ services:
     image: 169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/dissolution-api:latest
     expose:
       - 3001
+    environment:
```

Comment:
this file should be deleted now that we have docker-chs-development

Reply:
Removed.